ring/func/asyncio.py

```diff
@@ -71,6 +74,17 @@ def create_asyncio_factory_proxy(factory_table, *, support_asyncio):
     return fbase.create_factory_proxy(proxy_base, classifier, factory_table)
 
 
+def async_wrap(func):
+    @wraps(func)
+    async def run(*args, loop=None, executor=None, **kwargs):
```

Comment:
I don't know well about asyncio convention. So this is a question.

Does python users conventionally avoid to use parameter name `loop` and `executor`? Then the patch looks perfect and safe. Otherwise, what happens if the given user function `func` expects paramter `loop` and/or `executor`? In that case, would prefix these arguments a better idea than now?

Reply:
Thanks for your quick response. 
There is no convention to avoid use parameter name `loop` and `executor`. It can be problem in this library. It should be `event loop` and  `concurrent.futures.Executor`. I agree your idea. one of other idea is we don't accept loop as parameter. Prior to Python 3.6, asyncio.get_event_loop() was not guaranteed to return the event loop. So, It can be useful in Python <=3.6. But this code runs on >=3.8. 
I think `ring_executor` can be alternative of `executor`

Reference Links
- https://stackoverflow.com/questions/60312374/what-are-all-these-deprecated-loop-parameters-in-asyncio
- https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.run_in_executor

Reply:
Thank you for the explanation!